framework/helpers/BaseHtml.php

```diff
@@ -2311,6 +2311,12 @@ public static function getAttributeValue($model, $attribute)
             $value = $value->getPrimaryKey(false);
 
             return is_array($value) ? json_encode($value) : $value;
+        } elseif (version_compare(PHP_VERSION, '8.1.0') >= 0) {
```

Comment:
Is this necessary?

`$var instanceof \Undefined\Class\Fqcn` – it's correct expression in PHP

Reply:
I basically reused the same code snippet used on yii\db\Command around line 380 which includes the PHP version check before checking for the enum type. Trying to be consistent with that.

Reply:
I think we should use `PHP_VERSION_ID >= 80100` in both places - it should be more efficient and consistent with other places.